tsconfig.json

```diff
@@ -17,6 +17,7 @@
     "noFallthroughCasesInSwitch": true,
     "jsx": "react",
     "outDir": "./build",
+    "noEmit": true,
```

Comment:
Set tsc to not emit any javascript when it runs, since we don't use tsc for compiling but we get weird errors when running the tests if we have accidentally have generated some js code